contactsummary/js/src/contact-summary/controllers/keyDetails.js

```diff
@@ -26,6 +26,10 @@ define([
                 return Contract.getPrimary();
             }.bind(this))
             .then(function (response) {
+                if (_.isEmpty(response)) {
+                    return;
```

Comment:
I followed the convention (all of the fields in contact summary have empty defaults). I can add some default if needed.